pylint/testutils/_primer/primer_run_command.py

```diff
@@ -23,57 +22,48 @@
 class RunCommand(PrimerCommand):
     def run(self) -> None:
         packages: dict[str, list[OldJsonExport]] = {}
-        astroid_errors: list[Message] = []
-        other_fatal_msgs: list[Message] = []
+        fatal_msgs: list[Message] = []
         for package, data in self.packages.items():
-            messages, p_astroid_errors, p_other_fatal_msgs = self._lint_package(
-                package, data
-            )
-            astroid_errors += p_astroid_errors
-            other_fatal_msgs += p_other_fatal_msgs
+            messages, p_fatal_msgs = self._lint_package(package, data)
+            fatal_msgs += p_fatal_msgs
             packages[package] = messages
-        plural = "s" if len(other_fatal_msgs) > 1 else ""
-        assert not other_fatal_msgs, (
-            f"We encountered {len(other_fatal_msgs)} fatal error message{plural}"
-            " that can't be attributed to bleeding edge astroid alone (see log)."
-        )
         path = (
             self.primer_directory
             / f"output_{'.'.join(str(i) for i in sys.version_info[:3])}_{self.config.type}.txt"
         )
         print(f"Writing result in {path}")
         with open(path, "w", encoding="utf-8") as f:
             json.dump(packages, f)
+        # Assert that a PR run does not introduce new fatal errors
+        if self.config.type == "pr":
+            plural = "s" if len(fatal_msgs) > 1 else ""
+            assert (
+                not fatal_msgs
+            ), f"We encountered {len(fatal_msgs)} fatal error message{plural} (see log)."
 
     @staticmethod
-    def _filter_astroid_errors(
+    def _filter_fatal_errors(
```

Comment:
The failure that prompted me to open the issue clearly shows that `astroid-error` does not mean the error actually comes from `astroid` 😄 

Let's remove the distinction.